index.js

```diff
@@ -210,18 +184,16 @@ module.exports = {
       this.setGasPrice.bind(this)
     ], function (err) {
       if (err) return self.retryConnect(err, options, callback);
-      self.state.connection = {
-        http: self.rpc.nodes.local || self.rpc.nodes.hosted,
-        ws: self.rpc.wsUrl,
-        ipc: self.rpc.ipcpath
-      };
+      self.state.connection = true;
```

Comment:
What is the use case for the results?  When I made this change, I looked through the code in Augur.js and Augur UI and didn't see anything using the results other than for a truthy check.

If the only use case is for debugging, then as I mentioned elsewhere, my recommendation is to not include code in production that is only for debugging, especially when that information is readily available in the browser.  That being said, if the desired use case is in fact debug type logging then perhaps the best way to handle this is to have Transporter `console.log` directly when it is finished connecting with the endpoints/transports that it is connected to?  This could be put behind the `rpc.debug.broadcast` flag so it is off by default but can be turned on?

If there really is a need to have connection information propagated all the way up the chain that will take a bit of work as right now only Transporter knows what ethrpc is actually connected to (and all logic for switching/choosing is in Transporter).